### PR TITLE
DB connectors appear in the UI if they are configured

### PR DESCRIPTION
**Pull request by alejandraarri**
We should display only the connectors that are configured in the `app_config.yml`.

### Related issue
https://github.com/CartoDB/support/issues/2332#issuecomment-575101514
https://github.com/CartoDB/support/issues/2332

### CODE REVIEW

**Comment by alejandraarri** [comment]
Retest this please

File: app/helpers/frontend_config_helper.rb [hunk 1]
```diff
@@ -35,6 +35,10 @@ def frontend_config_hash(user = current_user)
       oauth_mechanism_bigquery:   Cartodb.get_config(:oauth, 'bigquery', 'oauth_mechanism'),
       arcgis_enabled:             Cartodb.get_config(:datasources, 'arcgis_enabled'),
       salesforce_enabled:         Cartodb.get_config(:datasources, 'salesforce_enabled'),
+      mysql_enabled:              Cartodb.get_config(:connectors, 'mysql', 'enabled'),
```
Comment (jesusbotella) [inline app/helpers/frontend_config_helper.rb]: Have we set default values for this in the platform repo? :)

Comment (alejandraarri) [reply app/helpers/frontend_config_helper.rb]: This is our platform config [file](https://github.com/CartoDB/cartodb-platform/blob/master/cookbooks/cartodb_app/templates/default/app_config.yml.erb#L563) and this is the on-premises config [file](https://github.com/CartoDB/onpremises/blob/master/templates/builder/app_config.yml#L263)

**Comment by alejandraarri** [comment]
Retest this please

**Pull request merged**